app/models/katello/concerns/user_extensions.rb

```diff
@@ -58,113 +33,9 @@ def cp_oauth_header
           { 'cp-user' => self.login }
         end
 
-        def default_locale
-          self.preferences_hash[:user][:locale] rescue nil
-        end
-
-        def default_locale=(locale)
-          self.preferences_hash[:user] = { } unless self.preferences_hash.key? :user
-          self.preferences_hash[:user][:locale] = locale
-        end
-
-        def legacy_mode
-          self.preferences_hash[:user][:legacy_mode] rescue nil
-        end
-
-        def legacy_mode=(use_legacy_mode)
-          self.preferences_hash[:user] = { } unless self.preferences_hash.key? :user
-          self.preferences_hash[:user][:legacy_mode] = ::Foreman::Cast.to_bool(use_legacy_mode)
-        end
-
-        def default_org
-          org_id = self.preferences_hash[:user][:default_org] rescue nil
-          if org_id && !org_id.nil? && org_id != "nil"
-            org = Organization.find_by_id(org_id)
-            return org if self.organizations.include?(org)
-          else
-            return nil
-          end
-        end
-
-        #set the default org if it's an actual org_id
-        def default_org=(org_id)
-          self.preferences_hash[:user] = { } unless self.preferences_hash.key? :user
-          if !org_id.nil? && org_id != "nil"
-            organization = Organization.find_by_id(org_id)
-            self.preferences_hash[:user][:default_org] = organization.id
-          else
-            self.preferences_hash[:user][:default_org] = nil
-          end
-        end
-
-        def subscriptions_match_system_preference
-          self.preferences_hash[:user][:subscriptions_match_system] rescue false
```

Comment:
I do not recall, I could dig through the old code but these aren't used anywhere that I could find.